### DISCUSSION TITLE
Why is num_nodes not updated in FixedPoints transform?

### DISCUSSION THREAD

**Discussion by ptoews** [pyg-team/pytorch_geometric#4387] [Q&A] [answered]
It changes the count of points in the data instance and even uses the `num_nodes` attribute for selection, but doesn't change its value after updating all items of the data instance. 

https://github.com/pyg-team/pytorch_geometric/blob/36a80d7cff6be594366549c556ba4be0b6f5bde4/torch_geometric/transforms/fixed_points.py#L34-L55

**Comment by rusty1s** [comment, marked as answer]
Thanks for reporting. I fixed it in https://github.com/pyg-team/pytorch_geometric/pull/4394.
